TWLight/applications/templates/applications/application_list_include.html

```diff
@@ -9,9 +9,15 @@
 {% for app in object_list %}
   <div class="row">
     <div class="col-xs-4 col-sm-2">
+    {% if app.partner.authorization_method == 0 %}
```

Comment:
when you refer to the `authorization_method` please use the choice label instead of the number for legibility. You can search for `partner.authorization_method` in html files for examples.